apps/marketplace/components/BuyerSpecialist/BuyerSpecialistEvaluationCriteriaStage.js

```diff
@@ -284,8 +285,9 @@ class BuyerSpecialistEvaluationCriteriaStage extends Component {
                           id={`nice_to_have_weighting_${i}`}
```

Comment:
`nice_to_have_weighting_ to `Desirable_criteria_(optional)_ to reflect the specialist flow 

id={`nice_to_have_weighting_${i}`} to id={`Desirable_criteria_(optional)_${i}`}